### PR TITLE
Restructure watch features

### PR DESCRIPTION
**Pull request by beroset**
Address part one of #176 by creating and using a json file that contains all watch models and features.  It also removes some unintended inconsistencies in the various install template files.

### CODE REVIEW

**Comment by FlorentRevest** [comment]
Yaay :) Super cool, thanks a lot, that's a great improvement!

Just a general thing first: "features.json" is a bit of a generic name, when navigating pages it wouldn't immediately be obvious what that means (it may sound like a website feature). Maybe watches-support.json would be better suited ?

And then a nit, don't feel that you have to fix it because this is already a great improvement and I'd merge it as is (with the new name) but I see you define the supported and experimental array in the main-install page, but while we are at it why not express this distinction in the json model and have a helper function look it up, this way it would really be effortless to add a new watch to the website :) 

Anyway, thanks again for improving this kind of stuff

**Comment by beroset** [comment]
There are more improvements that could be made, but this is now feature complete, tested, and includes the features table with all watches and all features.

**Comment by beroset** [comment]
Rebased with current master.

**Comment by beroset** [comment]
Revised commit based on matrix conversation -- revert to "Install" rather than "Watches".  Later we can add a "Watches" top layer to describe watches features.

File: pages/watches-support.json [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+        { "name":"Touch", "status":"good" },
+        { "name":"Microphone", "status":"bad" },
+        { "name":"Bluetooth", "status":"bad" },
+        { "name":"Compass", "status":"good" },
+        { "name":"Haptics", "status":"good" },
+        { "name":"USB", "status":"good" }
+      ]
+    },
+    {
+      "name":"narwhal",
```
Comment (docgalaxyblock) [inline pages/watches-support.json:220]: `narwhal` has a working compass

File: pages/watches-support.json [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+        { "name":"Compass", "status":"good" },
+        { "name":"Haptics", "status":"good" },
+        { "name":"USB", "status":"good" },
+        { "name":"WLAN", "status":"good" },
+        { "name":"Tilt-to-Wake", "status":"good" },
+        { "name":"Always-on-Display", "status":"good" }
+      ]
+    },
+    {
+      "name":"wren",
```
Comment (docgalaxyblock) [inline pages/watches-support.json:416]: tilt-to-wake works on `wren`

File: pages/watches-support.json [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+        { "name":"Bluetooth", "status":"good" },
+        { "name":"Haptics", "status":"good" },
+        { "name":"USB", "status":"good" },
+        { "name":"WLAN", "status":"good" },
+        { "name":"Heart Rate", "status":"good" },
+        { "name":"Tilt-to-Wake", "status":"good" }
+      ]
+    },
+    {
+      "name":"sparrow",
```
Comment (docgalaxyblock) [inline pages/watches-support.json:322]: `sparrow` has a working speaker

**Comment by MagneFire** [comment]
@docgalaxyblock Thanks for your feedback on this PR!
However, I think it might be better to leave these feature changes to a followup PR, once this is merged as this PR already contains quite a bit of  changes.
It's also the reason I marked my PR (https://github.com/AsteroidOS/asteroidos.org/pull/179) as draft, I'll update that one once this one is merged.

Of course let me know if you disagree with this! I just think it's important to prioritize the structure of the data rather than correcting the data with this PR too.

**Comment by FlorentRevest** [comment]
Awesome work, thanks a lot beroset!

And I agree with what MagneFire brought up, let's address these separately :)

**Pull request merged**